generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -1,21 +1,19 @@
-<%_ if (integrations.includes('docker')) { _%>
```

Comment:
Yes but currently, there is no documentation about this new sub generator!! See https://jhipster.github.io/setting-up-ci/
Sadly, this morning I didn't have time to work on it, before going to my work day...